### DISCUSSION TITLE
Is there a Visual Studio (not VSC) server?

### DISCUSSION THREAD

**Discussion by ay0ks** [coder/code-server#5237] [Q&A] [answered]
Hello, are there any implementation of Visual Studio (not VSC) for web?
Thanks in advance.

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Hmm, that _is_ an idea, but here are caveats:

- that app is heavy as heck
- it could be C/C++

VsCode was TypeScript, hence we could use it here, but unsure on VS.
